doc/MiniLanguageTutorial.md

```diff
@@ -1,6 +1,6 @@
 # Mini language tutorial
 
-[version of June 5th, 2020]
+[version of July 12th, 2021]
```

Comment:
Maybe better to take this out?  Since the file is tracked in git, the version information should be available from git--which will be more reliable than a manually maintained version date.

Reply:
Removed -- it's not particularly clarifying and it's easy to forget about it